experimental/go-dep/image/Dockerfile-stack

```diff
@@ -0,0 +1,40 @@
+FROM golang:1
+
+ENV APPSODY_MOUNTS=/:/project/user-app
+ENV APPSODY_DEPS=
+
+ENV APPSODY_WATCH_DIR=/project/user-app
+ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/vendor
+ENV APPSODY_WATCH_REGEX="^.*.(go|toml)$"
+
+ENV APPSODY_INSTALL="bash /vendor.sh"
+
+ENV APPSODY_RUN="go run ."
+ENV APPSODY_RUN_ON_CHANGE="go run ."
+ENV APPSODY_RUN_KILL=false
+
+ENV APPSODY_DEBUG="dlv debug --listen 0.0.0.0:9229 --headless ."
```

Comment:
Looks like the debugger is running but I'm just wondering what this `could not launch process` error is?
![Screenshot 2019-10-16 at 09 27 50](https://user-images.githubusercontent.com/45264227/66901877-73c8a280-eff7-11e9-9cc1-854c2eec15f9.png)

Reply:
Might be related to https://stackoverflow.com/questions/35827819/fork-exec-debug-operation-not-permitted. I'll test once I figure out how to tell appsody to start debugging with `--privileged`.

Reply:
`appsody debug -I` resolve the problem. I can update the README to reflect this requirement if that is suitable, thoughts?

Reply:
Have yet to verify but yes, if that's the issue then it should be documented in the README. Also do you mean `appsody debug -i` as in using it in interactive TTY mode?

Reply:
Some README's for stacks have a Known Issues section but since this isn't a major issue as debugging still works you could probably add this as a note.

Reply:
I tested the debug command with the `-i` flag and the `could not launch process` error is still there. However, even without the `-i` flag debugging still seems so work fine.